www/jedv00/sp2/search/index.html

```diff
@@ -0,0 +1,131 @@
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" type="text/css" href="../css/style.css" />
+        <link
+        rel="stylesheet"
+        href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+        crossorigin="anonymous"
+        />
+        <script
+        src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+        integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+        crossorigin="anonymous"
+        ></script>
+        <script
+        src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+        integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+        crossorigin="anonymous"
+        ></script>
+        <script
+        src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+        integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+        crossorigin="anonymous"
+        ></script>
+        <title>My library</title>
+    </head>
+    <body class="text-center">
+        <div class="container">
+            <div class="row">
+                <div class="col-7 d-flex justify-content-end">
+                    <h1>My library</h1>
+                </div>
+                <div class="col-5 d-flex justify-content-end ">
+                    <div class="row mt-3 mb-3 justify-content-md-center">
+                    <button type="button" class="btn btn-primary" onclick="relocate()">📚</button>

```

Comment:
bez inline javascript

Reply:
Mohu se jen zeptat proč? Jinak samozřejmě provedu :)

Děkuju

Reply:
stejný důvod jako u CSS, jinak jsou tam faktory jako údržba, čitelnost, debugování, testování, performance - browser cache, aj. a hlavně to jde proti některým principům co jste se naučil/naučíte na jiných předmětech ohledně vývoje SW